src/main/CxAuth.ts

```diff
@@ -48,6 +49,10 @@ export class CxAuth {
         if (cxScanConfig.baseUri !== null && cxScanConfig.baseUri !== '') {
             this.baseUri = cxScanConfig.baseUri;
         }
+
+        if (cxScanConfig.tenant !== null && cxScanConfig.tenant !== '') {
```

Comment:
@jay-nanduri  can we replace this !== null and !== '' for just if (cxScanConfig.tenant)  ?

Reply:
Added the changes as mentioned Pedro. Thank you